src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -31,11 +31,15 @@
 from itertools import pairwise
 
 import numpy as np
+from genai_perf.llm_inputs.llm_inputs import OutputFormat
 from genai_perf.tokenizer import AutoTokenizer
 from genai_perf.utils import load_json, remove_sse_prefix
 from rich.console import Console
 from rich.table import Table
 
+_OPENAI_CHAT_COMPLETIONS = OutputFormat.OPENAI_CHAT_COMPLETIONS
+_OPENAI_COMPLETIONS = OutputFormat.OPENAI_COMPLETIONS
```

Comment:
just import these, dont define them

Reply:
I did import `OutputFormat` (it's an enum). I just redefined since it was too verbose.